instrumentation-genai/opentelemetry-instrumentation-openai-v2/examples/zero-code/README.rst

```diff
@@ -1,4 +1,4 @@
-OpenTelemetry OpenAI Instrumentation Example
+OpenTelemetry OpenAI Zero-Code Instrumentation Example
```

Comment:
Just a nit: Might be worth it to include automatic installation instructions instead of having a requirements.txt in the spirit of zero-code?

`pip install opentelemetry-distro`
`opentelemetry-bootstrap -a install`